modules/aioseop_sitemap.php

```diff
@@ -3664,6 +4023,25 @@ function get_all_post_counts( $args ) {
 					$post_counts[ $post_type ] = $this->get_post_count( $count_args );
 				}
 			}
+
+			/**
```

Comment:
The arrays were documented incorrectly

```
/**
 * Filter to list additional post types and their post counts.
 *
 * @since ?
 *
 * @param array $post_counts {
 *      Key/value map where key is the post type and value is the count of how many posts are in the post type.
 *
 *      @type int $page Page count.
 *      @type int $post Post count.
 * }
 * @param array $args {
 *     Arguments to send to WP_Query.
 *
 *     @type string $post_status Status to query.
 *     @type array  $post_type {
 *         Post types to query.
 *         @type string $index Post Type slug.
 *     }
 */
```